src/modules/compliance/src/lib/ComplianceInterface.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef COMPLIANCE_H
```

Comment:
>H [](http://example.com/codeflow?start=19&length=1)

Name the C++ headers with .h extensions, not .hpp. We are not using .hpp in this project.